lib/OpenQA/Task/Asset/Limit.pm

```diff
@@ -45,6 +46,7 @@ sub _update_exclusively_kept_asset_size {
 
 sub _limit {
     my ($app, $job) = @_;
+    my $signal_guard = OpenQA::Task::SignalGuard->new($job);
```

Comment:
can we make it clearer what the effect of this signal guard is? Maybe just rename the variable to explain what it should ensure. If I understand https://github.com/os-autoinst/openQA/commit/8b96b710 correctly then this could be

```suggestion
    my $ensure_task_retry_on_termination_signal_guard = OpenQA::Task::SignalGuard->new($job);
```

is it?